### ISSUE TITLE
Re-write BatchCrossValidationReport

### ISSUE THREAD

**Issue by daxenberger** [dkpro/dkpro-tc#113]
Originally reported on Google Code with ID 113

```
Instead of aggregating measures (e.g. F1) from individual CV fold runs by averaging
them (this might create biased results), BatchCrossValidationReport should create an
overall confusion matrix which can subsequently be used to properly calculate F1 etc.
```

Reported by `daxenberger.j` on 2014-04-22 15:48:42
- **Blocked on**: #50

**Comment by Horsmann** [comment]
This is related to https://github.com/dkpro/dkpro-tc/issues/50 isn't ?

**Comment by daxenberger** [comment]
yes, related.

**Comment by Horsmann** [comment]
@daxenberger This is auto-solved with #50 ?!

**Comment by daxenberger** [comment]
not fully I guess. As far as I can see, the bach-level reports create an overall report across all calculated measures - but not a confusion matrix. Could be tracked in its own issue though.

**Comment by Horsmann** [comment]
Own issue exists since a while #399, I close this one then
